core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2583,6 +2586,14 @@ private void loadFileMetadataAndJournal(RpcContext rpcContext, LockedInodePath i
         ufsStatus = ufs.getFileStatus(ufsUri.toString());
       }
       ufsLength = ufsStatus.getContentLength();
+
+      if (isAclEnabled()) {
+        try {
+          acl = ufs.getAcl(ufsUri.toString());
+        } catch (UnimplementedException e) {
+          // ACL is unsupported or disabled in under file system.
```

Comment:
Should a warning be logged?

Reply:
We shouldn't use exceptions for control flow if we expect UFSes to not implement this.

Reply:
Now, I returned `null` when ACL is unsupported or disabled in UFS.